### PR TITLE
Add tests for macros modulations

### LINKED ISSUE #591: Macromodulation do not get their customs variable
The Macromodulation objects do not get their customs attribute correctly set on configuration parsing.

When testing an AttributeEror exception is raised because of the missing `mm.customs` !

### PR DESCRIPTION
**Pull request by mohierf**
Closes #591

### CODE REVIEW

File: alignak/objects/macromodulation.py [hunk 1]
```diff
@@ -64,25 +64,32 @@ class MacroModulation(Item):
...
+        'macromodulation_name':
+            StringProp(fill_brok=['full_status']),
+        'modulation_period':
+            StringProp(brok_transformation=to_name_if_possible, fill_brok=['full_status']),
     })
 
     running_properties = Item.running_properties.copy()
+    running_properties.update({
+        'customs':
+            StringProp(default={}, fill_brok=['full_status']),
```
Comment (Seb-Solon) [inline alignak/objects/macromodulation.py]: StringProp defailt to {} ? Better be None right? I guess you pasted this from elsewhere?

File: alignak/objects/macromodulation.py [hunk 2]
```diff
@@ -64,25 +64,32 @@ class MacroModulation(Item):
...
 
     macros = {}
 
     def get_name(self):
         """
-        Get the name of the timeperiod
+        Get the name of the macromodulation
 
-        :return: the timeperiod name string
+        :return: the macromodulation name string
```
Comment (Seb-Solon) [inline alignak/objects/macromodulation.py]: :laughing:

File: alignak/objects/macromodulation.py (hunk 1 above)
Comment (mohierf) [inline alignak/objects/macromodulation.py]: Yes, i copied/pasted ... from SchedulingItem or Contact that have exactly this awful code! I may give a try with a DictProp ...

File: alignak/objects/macromodulation.py (hunk 2 above)
Comment (mohierf) [inline alignak/objects/macromodulation.py]: I still fixed a bunch of this ... but I obviously forgot one :wink:

**Pull request merged**